src/utils/index.ts

```diff
@@ -79,3 +79,14 @@ export const safeJSONParse = (str: string) => {
     return null;
   }
 };
+
+export const formatAddress = (address?: string, n = 4) => {
```

Comment:
maybe you can use https://github.com/RabbyHub/Rabby/blob/develop/src/ui/utils/address.ts#L1 for instead?

Reply:
for sure, thanks for pointers